pkg/daemon/util/copybins.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"io"
+	"os"
+	"os/exec"
+	"path"
+)
+
+// override these for unit testing
+var defaultRookDir = "/usr/local/bin"
+var defaultTiniDir = "/"
+
+// CopyBinaries copies the "tini" and "rook" binaries to a shared volume at the target path.
+func CopyBinaries(target string) error {
```

Comment:
Yes. They got duplicated to a higher level. A follow-up PR should be to make the OSDs use this, or I could add that here as well if that seems more appropriate.

Reply:
Let's plan this for a future PR, as this will also require refactoring the nfs-ganesha daemons as well. Those are still using an init container and using the osd's copy-binary funcitonality but not via the `copybins` command.

Reply:
sounds ok since the duplication will be cleaned up soon